streams/us_tubi.m3u

```diff
@@ -10,53 +10,49 @@ https://live-news-manifest.tubi.video/live-news-manifest/csm/extlive/tubiprd01,F
 #EXTINF:-1 tvg-id="KCCIDT1.us",KCCI-TV News Des Moines IA (720p)
 https://live-news-manifest.tubi.video/live-news-manifest/csm/extlive/tubiprd01,Cloudfront-KCCI.m3u8
 #EXTINF:-1 tvg-id="KCPQDT1.us",FOX Q13 Seattle WA (KCPQ) (720p)
-https://csm-e-eb.csm.tubi.video/csm/extlive/tubiprd01,p-KCPQ-Monetizer.m3u8
+https://live-news-manifest.tubi.video/live-news-manifest/csm/extlive/tubiprd01,KCPQ.m3u8
```

Comment:
link redirects to TS playlist files, however master https://lnc-kcpq.tubi.video/index.m3u8 works.